subsys/net/ip/tcp2_priv.h

```diff
@@ -152,12 +173,17 @@ struct tcp { /* TCP connection */
 	enum tcp_state state;
 	u32_t seq;
 	u32_t ack;
+	u32_t last_seq;
 	union tcp_endpoint src;
 	union tcp_endpoint dst;
-	u16_t win;
+	u16_t recv_win;
+	u16_t send_win;
 	struct tcp_options recv_options;
 	struct k_delayed_work send_timer;
+	struct k_delayed_work data_timer;
 	sys_slist_t send_queue;
+	struct net_pkt *pending_data;
```

Comment:
Might it make conceptually clearer to rename the pending_data into send_data and pending_len to unack_len? Would be even possible to eliminate pending_len?

Reply:
Sure, the variables can be renamed.

Not sure if we can eliminate pending_len, will investigate.